test/build_tf_cc.sh

```diff
@@ -57,7 +57,7 @@ pushd "${TF_ROOT}"
 ./configure
 
 # Build the TensorFlow C++ Library 
-bazel build --config=opt ${COPT} --config=mkl //tensorflow:libtensorflow_cc.so
+bazel build --config=opt ${COPT} //tensorflow:libtensorflow_cc.so
 
```

Comment:
We are removing mkl build?

Reply:
Yes - that's because when we use mkl - some TF ops go to MKL while others go to Eigen - resulting in failures due to precision issues. We shouldn't be testing with MKL (unless we are comparing performance) - so I removed it.